README.md

```diff
@@ -81,10 +81,7 @@ The following outlines our release process:
 1.  Checkout a new branch.
 2.  Bump package version in `packages/gestalt/package.json` & update `CHANGELOG.md`.
 3.  Open a pull request with the new version and land that in master.
-4.  Once the version is bumped in master, checkout that commit locally.
-5.  Run `npm login` using your npm username and password.
-6.  Run the release script from the root directory of the project `./scripts/publish.js` to publish the tag, npm package, and docs.
-7.  Draft a new release from the tag at https://github.com/pinterest/gestalt/releases.
+4.  Draft a new release from the tag at https://github.com/pinterest/gestalt/releases. GitHub Actions will automatically publish to npm and GitHub pages.
```

Comment:
@btraut Thank you for updating these - wanted to make sure everything worked as expected. Perhaps one thing to mention in a follow up is that version tags should start with `v`: https://github.com/pinterest/gestalt/blob/7f75e2a022b528364d274d1b4e312c5efb7ac74a/.github/workflows/publish.yml#L6